integrations/examples/exit-early.md

```diff
@@ -156,6 +177,19 @@ import_object.register(
 )
 ```
 {% endtab %}
+
+{% tab title="PHP" %}
+```php
+function earlyExit() {
+    throw new ExitCode(1);
+}
+
+$funcType = Wasm\Type\FuncType::new(new Wasm\Vec\ValType(), new Wasm\Vec\ValType());
+$func = Wasm\Module\Func::new($store, $funcType, 'earlyExit');
```

Comment:
Side question: why `Func` is inside `Module` ?
Funcs can be used without a module (they just need a store)

Reply:
I tried to be as close as possible to the spec when naming/organizing things: https://webassembly.github.io/spec/core/syntax/modules.html

My main goal was to not put too much things in the `Wasm` namespace directly. What I can do, is change from `Wasm\Module` to `Wasm\External` to replicate what we have in Rust.

Reply:
Fixed.

_See wasmerio/wasmer-php#141_